.mailmap

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024, The Khronos Group Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+Rylie Pavlik <rylie.pavlik@collabora.com>
+Rylie Pavlik <rylie.pavlik@collabora.com> <ryan.pavlik@collabora.com>
+
+Jon Leech <devrel@oddhack.org>
+Jon Leech <devrel@oddhack.org> <oddhack@sonic.net>
```

Comment:
I'd prefer not to have my Sonic address in here - it's my primary email address and I protect it to the extent possible. The oddhack.org addresses forward there, eventually.